universal-application-tool-0.0.1/app/views/admin/questions/QuestionEditView.java

```diff
@@ -210,6 +210,7 @@ private ContainerTag multiOptionQuestionField(QuestionForm questionForm) {
           FieldWithLabel.number()
               .setFieldName("nextAvailableId")
               .setValue(((MultiOptionQuestionForm) questionForm).getNextAvailableId())
+              .setScreenReaderText("next id, hidden")
```

Comment:
(question): should hidden fields have aria lables?

Reply:
eh, probably not valuable.  I added it so that the asserts I was testing with would pass.  Removed.